packages/strapi-design-system/src/Button/Button.stories.mdx

```diff
@@ -1,70 +1,155 @@
 
 # Button
 
-This is the doc of the `button` component
+<Description markdown="Button description..." />
 
-## Variants
+[View source](https://github.com/strapi/design-system-experiments/tree/develop/packages/strapi-design-system/src/Button)
 
-Description...
+## Imports
 
-<Canvas>
-  <Story name="variants">
-    {['default', 'secondary', 'tertiary', 'success', 'danger', 'success-light', 'danger-light'].map((variant) => (
-      <Box key={variant} paddingBottom={2}>
-        <Button onClick={(e) => console.log(e)} variant={variant}>
-          {variant}
-        </Button>
-      </Box>
-    ))}
-  </Story>
-</Canvas>
+```js
+import { Button } from '@strapi/design-system/Button';
+```
 
-## Disabled
+## Usage
 
-Description...
-
-<Canvas>
-  <Story name="disabled">
+<Canvas withSource="open">
+  <Story name="default" parameters={{ docs: { source: { code: '<Button>Default button</Button>' } } }}>
     <Box paddingBottom={2}>
-      <Button disabled>Disabled</Button>
+      <Button variant="default">Default</Button>
     </Box>
-    <Button onClick={(e) => console.log(e)} startIcon={<ContentIcon />} disabled>
-      Disabled
-    </Button>
   </Story>
 </Canvas>
 
-## With icon
+### Button sizes
+
+Use the size prop to change the size of the button. You can set the value to `S` or `L`.
+
+<Canvas withSource="open">
+  <Story
+    name="sizes"
+    parameters={{
+      docs: {
+        source: {
+          code: `
+<Button size="S">Small</Button>
+<Button size="L">Large</Button>
+`,
+        },
+      },
+    }}
+  >
+    <Row>
+      <Box paddingRight={1}>
+        <Button size="S">Small</Button>
+      </Box>
+      <Button size="L">Large</Button>
+    </Row>
+  </Story>
+</Canvas>
 
-Description...
+### Button variants
+
+Use the variant prop to change the visual style of the Button.
+You can set the value to `default`, `secondary`, `tertiary`, `success`, `danger`, `success-light`, or `danger-light`.
+
+<Canvas withSource="open">
+  <Story
+    name="variants"
+    parameters={{
+      docs: {
+        source: {
+          code: `
+<Button variant='default'>Default</Button>
+<Button variant='secondary'>Secondary</Button>
+<Button variant='tertiary'>Tertiary</Button>
+...
+  `,
+        },
+      },
+    }}
+  >
+    <Row>
+      {['default', 'secondary', 'tertiary', 'success', 'danger', 'success-light', 'danger-light'].map((variant) => (
+        <Box key={variant} paddingRight={2}>
+          <Button onClick={(e) => console.log(e)} variant={variant}>
+            {variant}
+          </Button>
+        </Box>
+      ))}
+    </Row>
+  </Story>
+</Canvas>
 
-<Canvas>
-  <Story name="with-icon">
-    <Box paddingBottom={2}>
-      <Button startIcon={<ContentIcon />}>Content type</Button>
+### Button with icon
+
+You can add start and end icons to the Button component using the `startIcon` and `endIcon` props respectively.
+
+> Note: The startIcon and endIcon prop values should be react elements NOT strings.
```

Comment:
I think this sentence assumes knowledge that people were using strings before. Is this relevant?